source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +354,122 @@ the steps described in :ref:`debugging-deployer-issues`
   Changing or deleting a processed commit file could cause unchanged files to be indexed again and
   it should be done as a last resort in case of errors.
 
+^^^^^^^^^^^^^^^^^
+Document Indexing
+^^^^^^^^^^^^^^^^^
+Crafter Deployer indexes documents via the following configured items:
+
+- A full text index of any document that has a mimetype that matches the configured list of mimetypes.
+- Indexing of any remote document that matches the configured list of remote documents pattern
+- Indexing of jacketed documents with anything that matches the configured pattern.
```

Comment:
- Full-text indexing of text-based documents with a CrafterCMS content wrapper [reference what that is with a link and another link to the jacket example below].